src/syscalls/fs.rs

```diff
@@ -98,25 +99,44 @@ impl Filesystem {
 		&'a self,
 		path: &'b str,
 	) -> Result<(&'a (dyn PosixFileSystem + Send), &'b str), FileError> {
-		// assert start with / (no pwd relative!), split path at /, look first element. Determine backing fs. If non existent, -ENOENT
-		if !path.starts_with('/') {
-			warn!("Relative paths not allowed!");
-			return Err(FileError::ENOENT());
-		}
-		let mut pathsplit = path.splitn(3, '/');
-		pathsplit.next(); // always empty, since first char is /
-		let mount = pathsplit.next().unwrap();
-		let internal_path = pathsplit.next().unwrap(); //TODO: this can fail from userspace, eg when passing "/test"
+		if path.starts_with('/') {
+			let mut pathsplit = path.splitn(3, '/');
+			pathsplit.next(); // empty, since first char is /
+
+			let mount = pathsplit.next().unwrap();
+			let internal_path = pathsplit.next().unwrap();
+			if let Some(fs) = self.mounts.get(mount) {
+				return Ok((fs.deref(), internal_path));
+			}
 
-		if let Some(fs) = self.mounts.get(mount) {
-			Ok((fs.deref(), internal_path))
+			info!(
+				"Trying to open file on non-existing mount point '{}'!",
+				mount
+			);
```

Comment:
Is info fitting here or should that be a warning? I would expect that to be a user configuration error.